client/src/components/WebAuthn/Register.js

```diff
@@ -117,13 +117,15 @@ class Register extends Component {
         <p>
           {i18n._t('MFAWebAuthnRegister.DESCRIPTION', fallbacks['MFAWebAuthnRegister.DESCRIPTION'])}
 
-          <a
-            href={method.supportLink}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            {i18n._t('MFAWebAuthnRegister.HELP', fallbacks['MFAWebAuthnRegister.HELP'])}
-          </a>
+          {supportLink &&
+            <a
+              href={supportLink}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {i18n._t('MFAWebAuthnRegister.HELP', fallbacks['MFAWebAuthnRegister.HELP'])}
```

Comment:
As with the other PRs, I think you can use `method.supportText` and leave this as a fallback